indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -132,6 +132,12 @@
                                    data-update="#contribution-list"
                                    data-method="DELETE"
                                    data-href="{{ url_for('.manage_contrib_rest', contrib) }}"></a>
+                                <a href="#" class="icon-lock"
+                                   data-href="{{ url_for('.manage_contrib_protection', contrib) }}"
+                                   data-method="GET"
```

Comment:
Indeed, if `data-method` is not specified, then `GET` is the default HTTP method